src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx

```diff
@@ -69,89 +71,99 @@ const ReviewChangeRequestsView: React.FC<ReviewChangeRequestViewProps> = ({
 
   const renderProposedSolutionModal: (scr: StandardChangeRequest) => JSX.Element = (scr: StandardChangeRequest) => {
     return (
-      <Dialog open={modalShow} onClose={onHide} style={{ color: 'black' }}>
-        <DialogTitle className={'font-weight-bold'}>{`Review Change Request #${cr.crId}`}</DialogTitle>
-        <DialogContent
-          sx={{
-            '&::-webkit-scrollbar': {
-              display: 'none'
-            }
-          }}
-        >
-          <Typography sx={{ paddingBottom: 1 }}>{'Select Proposed Solution'}</Typography>
-          <Box
+      <>
+        <Toast open={showAlert} severity="error" message={'Please select a proposed solution!'} stateSetter={setShowAlert} />
+        <Dialog open={modalShow} onClose={onHide} style={{ color: 'black' }}>
```

Comment:
Could you make the Toast duration a lil longer, also just so this PR demonstrates how to use the duration prop